cumulusci/tasks/create_package_version.py

```diff
@@ -266,6 +267,20 @@ def _run_task(self):
         self.logger.info(f"  Version Number: {self.return_values['version_number']}")
         self.logger.info(f"  Dependencies: {self.return_values['dependencies']}")
 
+    def _get_spv_where_clause(self, spv_id: str) -> str:
```

Comment:
It's a bummer to have this code repeated in two different tasks. What do you think about breaking it out into a utility method, `format_subscriber_package_version_where_clause(spv_id: str, install_key: Optional[str]) -> str`?